wsbroadcastserver/wsbroadcastserver.go

```diff
@@ -265,7 +273,7 @@ func (s *WSBroadcastServer) StartWithHeader(ctx context.Context, header ws.Hands
 					}
 					requestedSeqNum = arbutil.MessageIndex(num)
 				} else if headerName == HTTPHeaderCloudflareConnectingIP {
```

Comment:
Let's change these headers to case insensitive compares (either with `strings.EqualFold` or by setting headerName to the result of `textproto.CanonicalMIMEHeaderKey` on itself) since we're changing the constants' casing.

Reply:
Done